packages/core/package.json

```
@@ -72,7 +72,8 @@
 		"tslib": "^2.5.0",
 		"universal-cookie": "^4.0.4",
 		"zen-observable-ts": "0.8.19",
-		"rxjs": "^7.8.1"
+		"rxjs": "^7.8.1",
+		"js-cookie": "3.0.5"
```

Do we need to add another dependency? could be possible to not have this?

---

Yeah I think we discussed this in the design already. Pretty much `js-cookie` is to handle the overwhelming cookies operations. The package is pretty light weight and well supported: https://www.npmjs.com/package/js-cookie

---

Looks good but we might want to loosen up the dependency in-case customers have this in their graph as well

```suggestion
		"js-cookie": "^3.0.5"
```